test/unit/network_spec.js

```diff
@@ -69,11 +69,10 @@ describe('network', function() {
     // The test is valid for FF only: the XHR has support of the
     // 'moz-chunked-array' response type.
     // TODO enable for other browsers, e.g. when fetch/streams API is supported.
-    var m = /Mozilla\/5.0.*?rv:(\d+).*? Gecko/.exec(userAgent);
-    if (!m || m[1] < 9) {
-      expect(true).toEqual(true);
-      done();
-      return;
+    var isFirefoxWithMozChunkedEncodingSupport = /Mozilla\/5.0.*?rv:(\d+).*? Gecko/.exec(userAgent);
```

Comment:
The use of `is` in the variable name suggest that it should contain a boolean, however that isn't what's happening here. What you probably want is to adjust which regexp function you're using:
```javascript
let isFirefoxWithMozChunkedEncodingSupport =
    /Mozilla\/5.0.*?rv:(\d+).*? Gecko/.exec(userAgent);
```